pkg/account/deployer/client.go

```diff
@@ -181,7 +182,7 @@ func (d *accountManagementClient) upsertGroup(ctx context.Context, groupId strin
 	}
 
 	if len(existingGroups) > 1 { // shouldn't happen
-		log.Warn("Updating multiple policies with name %s. Updating group with UUID %q", group.Name, existingGroups[0].GetUuid(), group.Name)
+		logr.FromContextOrDiscard(ctx).V(1).Info("Updating multiple policies with name " + group.Name + ". Updating group with UUID " + existingGroups[0].GetUuid())
```

Comment:
You use V(1) for 'warn' here, and for 'debug' on other lines - please double check what the right verbosity levels are for logr calls